File: questPresentation/src/main/java/ru/nekit/android/qls/quest/view/mediator/types/metrics/MetricsQuestContentMediator.java
package ru.nekit.android.qls.quest.view.mediator.types.metrics;

import android.support.annotation.NonNull;
import android.view.View;
import android.widget.EditText;

import ru.nekit.android.qls.quest.QuestContext;
import ru.nekit.android.qls.quest.formatter.IQuestTextContentFormatter;
import ru.nekit.android.qls.quest.formatter.MetricsQuestContentFormatter;
import ru.nekit.android.qls.quest.view.mediator.types.simpleExample.SimpleExampleQuestContentMediator;

public class MetricsQuestContentMediator extends SimpleExampleQuestContentMediator {

    @NonNull
    protected IQuestTextContentFormatter createFormatter() {
        return new MetricsQuestContentFormatter();
    }

    @Override
    public void onCreate(@NonNull QuestContext questContext) {
        super.onCreate(questContext);
        mViewHolder.alternativeAnswerInput.setVisibility(View.GONE);
    }

    @Override
    public EditText getAnswerInput() {
        return null;
    }

}

File: questPresentation/src/main/java/ru/nekit/android/qls/quest/view/mediator/types/metrics/MetricsQuestContentMediator.kt
package ru.nekit.android.qls.quest.view.mediator.types.metrics

import android.view.View
import android.widget.EditText
import ru.nekit.android.qls.domain.model.quest.Quest

import ru.nekit.android.qls.quest.QuestContext
import ru.nekit.android.qls.quest.formatter.IQuestTextContentFormatter
import ru.nekit.android.qls.quest.formatter.MetricsQuestContentFormatter
import ru.nekit.android.qls.quest.view.mediator.types.simpleExample.SimpleExampleQuestContentMediator

//ver 1.0
class MetricsQuestContentMediator : SimpleExampleQuestContentMediator() {

    override val answerInput: EditText?
        get() = null

    override fun createFormatter(): IQuestTextContentFormatter {
        return MetricsQuestContentFormatter()
    }

    override fun onCreate(questContext: QuestContext, quest: Quest) {
        super.onCreate(questContext, quest)
        viewHolder.alternativeAnswerInput.visibility = View.GONE
    }

}
